train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -59,6 +59,7 @@ private void enforceSpeedLimit() {
 	@Override
 	public void setJoystickPosition(int joystickPosition) {
 		this.stepperChange = joystickPosition;
+		followSpeed();
```

Comment:
implementation is not done, please implement a timer in the corresponding method